core/src/test/java/org/elasticsearch/test/ESBackcompatTestCase.java

```diff
@@ -141,6 +143,20 @@ private static Path backwardsCompatibilityPath() {
         return builder;
     }
 
+    @Override
+    public Settings indexSettings() {
+        // IndexMetaData.SETTING_DATA_PATH looks like it breaks backwards compatibility tests
```

Comment:
wonder whether it's ok that it breaks tests, or whether we should dig... I think it shouldn't break things in theory...